docs/errors.md

```diff
@@ -129,17 +129,6 @@ The algorithm used for generating your key isn't listed as supported. You can ge
   jose newkey -s 512 -t oct -a HS512
 ```
 
-If you are unable to use an HS512 key (for example to interoperate with other services) you can define what is supported using
-
-```
-  jwt: {
-    signingKey: {"kty":"oct","kid":"--","alg":"HS256","k":"--"},
-    verificationOptions: {
-      algorithms: ["HS256"]
-    }
-  }
-```
-
```

Comment:
Configuring it like this is not longer possible, however I believe you could achieve this through the `encode` function? Something like:
```ts
  jwt: {
    secret: process.env.SECRET,
    encode: async ({ secret, token }) => {
      return jwt.sign({...token, userId: token.id}, secret, {
        algorithm: "HS256",
        kty: "oct",
        kid: "––",
        k: "––"
      });
    },
```

Reply:
Correct. `jwt.sign` has to be imported from something like `jose` or `jsonwebtoken`.

I think we can safely drop this section tbh, we are not recommending for anyone to use the NextAuth.js issued JWT outside of their app, so these changes are only necessary in very rare cases.

We generally don't want people to even use `encode`/`decode`, it's there for heavy users who hopefully understand its purpose.